CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ These libraries have gone through code quality checks including for [GNU Complex
    - [FreeRTOS/coreJSON](https://github.com/FreeRTOS/JSON) for JSON parser library
    - [aws/device-shadow-for-aws-iot-embedded-sdk](https://github.com/aws/device-shadow-for-aws-iot-embedded-sdk) for AWS IoT Shadow service client library.
 
-- With this release, we are introducing a [date-based versioning scheme](LINK TO README EXPLAINING VERSIONING ONCE UPDATED).
+- With this release, we are introducing a [date-based versioning scheme](README.md#versioning).
```

Comment:
This link is pointing to README.md of master branch and there is no #versioning link present. How is this supposed to work?

Reply:
It points to the `README` in the same branch of the CHANGELOG file instead of master branch.